packages/workbox-core/INTERNAL/models/messages/messageGenerator.mjs

```diff
@@ -0,0 +1,29 @@
+import messages from './messages.mjs';
+import core from '../../../index.mjs';
+
+const fallback = (code, ...args) => {
+  let msg = `${code}`;
+  if (args.length > 0) {
+    msg += ` :: ${JSON.stringify(args)}`;
+  }
+  return msg;
+};
+
+const generatorFunction = (code, ...args) => {
+  const message = messages[code];
+  if (!message) {
+    core.INTERNAL.logHelper.warn(`Unable to find message for code '${code}'`);
+    return fallback(code, ...args);
+  }
+
+  try {
+    return message(...args);
+  } catch (err) {
+    return fallback(code, ...args);
```

Comment:
Can you add a

```js
console.error(`Unable to generate full error message: ${err}`);
```

here so that we can figure out why `message(...args)` failed?